services/web/server/src/simcore_service_webserver/db_models.py

```diff
@@ -9,11 +9,13 @@
     UserStatus,
     api_keys,
     confirmations,
+    folders,
     group_classifiers,
     groups,
     products,
     projects,
     scicrunch_resources,
+    study_folder,
```

Comment:
rename to projects. let's not have study anymore.

Reply:
renamed to folder_to_project